openmp/libompd/test/ompt_plugin.h

```diff
@@ -169,7 +169,7 @@ void collectTaskData(omp_t_data_t * data)
 	data->omp_parallel = omp_in_parallel();
 	data->omp_final = omp_in_final();
 	data->omp_dynamic = omp_get_dynamic();
-	data->omp_nested = omp_get_nested();
+	data->omp_nested = omp_get_max_active_levels();
```

Comment:
I think, it would be better to just remove this at all? Otherwise it should be:
```suggestion
	data->omp_nested = omp_get_max_active_levels()>1;
```

Reply:
Modified the same.